lib/gym/xcodebuild_fixes/swift_fix.rb

```diff
@@ -6,10 +6,9 @@ class << self
       def swift_library_fix
         require 'fileutils'
 
-        ipa_swift_frameworks = Dir["#{PackageCommandGenerator.appfile_path}/Frameworks/libswift*"]
         Helper.log.info "Checking for Swift framework" if $verbose
+        return unless system %(unzip -l '#{PackageCommandGenerator.ipa_path}' "#{PackageCommandGenerator.appfile_path}/Frameworks/libswift*" > /dev/null)
```

Comment:
Could we do this within Ruby without so much shell magic? Or would that require a lot of extra lines?